### ISSUE TITLE
Documentation of the configuration settings

### ISSUE THREAD

**Issue by tobilg** [tidwall/tile38#289]
Thanks for this great piece of software! I'd really like to see the configuration parameters and methods documented in a way that make a start a bit easier (without having to look at the source code).

If I can provide a PR or help in another way let me know.

**Comment by tidwall** [comment]
I agree. I think this may be a good candidate for a topic page on http://tile38.com/documentation.

I would very much appreciate your help. Right now the tile38.com content is being managed at https://github.com/tile38/tile38.github.io.

**Comment by tobilg** [comment]
Could you point me to the relevant portions in the code from your POV? I can try to do a PR next week then... Thanks!

**Comment by umpc** [comment]
Hi @tobilg. Would this be the file that you're looking for?
https://github.com/tidwall/tile38/blob/master/controller/config.go

**Comment by SvenDowideit** [comment]
I'm guessing that's now https://github.com/tidwall/tile38/blob/master/internal/server/config.go ?
